airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -82,11 +83,11 @@ def get_job_group(self, job_id: int, embed: str, include_deleted: bool) -> Dict[
         A job group is a job that is executed from a specific node in a flow.
 
         :param job_id The ID of the job that will be fetched.
-        :type job_id: int
+        :type job_id: int.
         :param embed Comma-separated list of objects to pull in as part of the response.
-        :type embed: string
+        :type embed: string.
         :param include_deleted if set to "true", will include deleted objects.
-        :type include_deleted: bool
+        :type include_deleted: bool.
```

Comment:
```suggestion
        :type include_deleted: bool
```
We avoid dots in type docs

Reply:
I saw documentation with full stops and I was wondering if I should add full stops in mine.
Ok, I will remove full stops.

Reply:
Looking for `:type .*: .* \.` in `airflow/providers` returns no records